Alloy/commands/compile/index.js

```diff
@@ -706,7 +713,7 @@ function parseAlloyComponent(view, dir, manifest, noView, fileRestriction) {
 	}
 	var cCode = CU.loadController(files.CONTROLLER);
 	template.parentController = (cCode.parentControllerName !== '') ?
-		cCode.parentControllerName : "'BaseController'";
+    	cCode.parentControllerName : CU[CONST.DOCROOT_BASECONTROLLER_PROPERTY] || "'BaseController'";
```

Comment:
I checked in a fix.  Let me know if that is not the correct format and I can resubmit.  Thanks!